editor/src/components/inspector/sections/layout-section/conditional-section.tsx

```diff
@@ -433,21 +445,35 @@ const BranchRow = ({
           whiteSpace: 'nowrap',
         }}
       >
-        <LayoutIcon
-          key={`layout-type-${label}`}
-          navigatorEntry={navigatorEntry}
-          color='main'
-          warningText={null}
-        />
-        <span
-          data-testid={
-            conditionalCase === 'true-case'
-              ? ConditionalsControlBranchTrueTestId
-              : ConditionalsControlBranchFalseTestId
-          }
-        >
-          {getNavigatorEntryLabel(navigatorEntry, label)}
-        </span>
+        {isSlot ? (
+          <div
+            style={{
+              padding: '0px 6px',
+              textTransform: 'lowercase',
+              color: colorTheme.fg7.value,
```

Comment:
you could use a `getControlStyles('unset').mainColor` here, to track the changes to control styles in the future!